test/test_helper.rb

```diff
@@ -6,95 +6,34 @@
 require 'action_controller/test_case'
 require 'action_controller/railtie'
 require 'active_support/json'
-require 'minitest/autorun'
 require 'fileutils'
+FileUtils.mkdir_p(File.expand_path('../../tmp/cache', __FILE__))
+
+require 'minitest/autorun'
 # Ensure backward compatibility with Minitest 4
 Minitest::Test = MiniTest::Unit::TestCase unless defined?(Minitest::Test)
 
-require "capture_warnings"
-@capture_warnings = CaptureWarnings.new(fail_build = false)
```

Comment:
factored these out into `test/support/*.rb` so it's easier to see what's being configured